astronomer/providers/amazon/aws/sensors/emr.py

```diff
@@ -0,0 +1,51 @@
+from typing import Any, Dict
+
+from airflow import AirflowException
+from airflow.providers.amazon.aws.sensors.emr import EmrStepSensor
+
+from astronomer.providers.amazon.aws.triggers.emr import EmrStepSensorTrigger
+
+
+class EmrStepSensorAsync(EmrStepSensor):
+    """
+    Async (deferring) version of EmrStepSensor
+
+    Asks for the state of the step until it reaches any of the target states.
+    If it fails the sensor errors, failing the task.
```

Comment:
If it fails the sensor errors, failing the task >> Please reword it so that it is more meaningful..something like ... "If the sensor errors out, then the task will fail"